include/mgp.hpp

```diff
@@ -985,6 +953,7 @@ class Value {
   friend class Result;
 
   explicit Value(mgp_value *ptr);
+  explicit Value(mgp_value **ptr);
```

Comment:
I would remove this and force the caller to dereference the pointer.

Reply:
Can you check `inline Value::Value(mgp_value **ptr) : ptr_(*ptr) {}` for more extensive comment why I used this? If that doens't make sense I can change it.